protocol-units/da/movement/celestia/light-node/src/v1/passthrough.rs

```diff
@@ -19,44 +21,29 @@ use movement_da_light_node_proto::light_node_service_server::LightNodeService;
 use movement_da_light_node_proto::*;
 
 use crate::v1::LightNodeV1Operations;
-use ecdsa::{
-	elliptic_curve::{
-		generic_array::ArrayLength,
-		ops::Invert,
-		point::PointCompression,
-		sec1::{FromEncodedPoint, ModulusSize, ToEncodedPoint},
-		subtle::CtOption,
-		AffinePoint, CurveArithmetic, FieldBytesSize, PrimeCurve, Scalar,
-	},
-	hazmat::{DigestPrimitive, SignPrimitive, VerifyPrimitive},
-	SignatureSize, SigningKey,
-};
+use movement_celestia_light_node_signer::Signer;
+use movement_signer::{cryptography::Curve, Digester, Signing, Verify};
+use movement_signer_loader::identifiers::LoadedSigner;
 
 #[derive(Clone)]
-pub struct LightNodeV1<C>
+pub struct LightNodeV1<O, C>
 where
-	C: PrimeCurve + CurveArithmetic + DigestPrimitive + PointCompression,
-	Scalar<C>: Invert<Output = CtOption<Scalar<C>>> + SignPrimitive<C>,
-	SignatureSize<C>: ArrayLength<u8>,
-	AffinePoint<C>: FromEncodedPoint<C> + ToEncodedPoint<C> + VerifyPrimitive<C>,
-	FieldBytesSize<C>: ModulusSize,
+	O: Signing<C> + Send + Sync + Clone,
+	C: Curve + Send + Sync + Clone,
 {
 	pub config: Config,
 	pub celestia_namespace: Namespace,
 	pub default_client: Arc<Client>,
 	pub verifier: Arc<
 		Box<dyn VerifierOperations<CelestiaBlob, IntermediateBlobRepresentation> + Send + Sync>,
 	>,
-	pub signing_key: SigningKey<C>,
+	pub signer: Arc<Signer<O, C>>,
```

Comment:
So we're punting the blob signature curve choice as a generic parameter up to here, but the `VerifierOperations` trait does not fix it, so...